app/_layout.tsx

```diff
@@ -98,26 +110,28 @@ export default function RootLayout() {
 
   return (
     <SWRConfig value={swrConfiguration}>
-      <ThemeProvider value={isDarkColorScheme ? DARK_THEME : LIGHT_THEME}>
-        <StatusBar style={isDarkColorScheme ? "light" : "dark"} />
-        <SafeAreaView
-          className="w-full h-full bg-background"
-          edges={["left", "right", "bottom"]}
-        >
-          <UserInactivityProvider>
-            <SessionProvider>
-              <Slot />
-            </SessionProvider>
-          </UserInactivityProvider>
-          <Toast
-            config={toastConfig}
-            position="bottom"
-            bottomOffset={140}
-            topOffset={140}
-          />
-          <PortalHost />
-        </SafeAreaView>
-      </ThemeProvider>
+      <NotificationProvider>
```

Comment:
Can we conditionally include this based on an env variable? that way expo can still run without errors in dev mode

Reply:
Done!